src/pages/contact.md

```diff
@@ -29,3 +29,18 @@ There are a few different channels you can reach us based on your needs:
 - **Feedback and support** can be submitted or requested via JIRA by following our [Enterprise support process](https://docs.camunda.org/enterprise/support/). All users can also find feedback and support options in the Help Center or [Camunda community forum](https://forum.camunda.io/).
 
 - For sales inquiries, information about Camunda 8 performance and benchmarking, or anything not listed above, use our [Contact Us](https://camunda.com/contact/) form.
+
+## Locating Camunda 8 credentials
+
+Need assistance locating your Camunda 8 credentials? You can obtain these credentials from Camunda by submitting a **Help Request**. To do this, take the following steps:
+
+1. Log in to [Jira](https://jira.camunda.com/secure/Dashboard.jspa).
+2. Click **Create** in the navigation bar at the top of the page. This launches a **Create Issue** pop-up.
+3. In the **Issue Type** field, select **Help Request**.
+4. In the **Help Request Type** field, click the option that reads **I need the credentials for downloading Camunda 7**.
```

Comment:
This was my biggest question, and what leads me to think is this the correct guidance?

The field is labeled **I need the credentials for downloading Camunda 7**, but should this just read **I need the credentials for downloading Camunda**, or should we have a separate field for C8?

Reply:
At the moment the field is labeled "_I need the credentials for downloading Camunda_". The documentation on Camunda 7 is not updated.

Reply:
This is how it looks from customer perspective:
<img width="287" alt="credentials" src="https://github.com/camunda/camunda-docs/assets/111515967/434c4481-3b8d-4c04-ada7-2b4d8f175136">

Reply:
Cool -- I will update!